src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/UsersController.kt

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.controller
+
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.UsersApiDelegate
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.User
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.problem.ForbiddenProblem
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.problem.NotFoundProblem
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.results.AuthorisableActionResult
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.UserService
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.UserTransformer
+
+@Service
+class UsersController(
+  private val userService: UserService,
+  private val userTransformer: UserTransformer
+) : UsersApiDelegate {
+
+  override fun usersIdGet(id: java.util.UUID): ResponseEntity<User> {
```

Comment:
Would move the fully qualified type name here to an import instead - if you hit Alt + Enter (Cmd + Enter on Mac?) then "Add import for 'java.util.UUID'" IntelliJ will do it for you